Sources/SwifterSwift/Foundation/URLSessionExtensions.swift

```diff
@@ -0,0 +1,33 @@
+// URLSessionExtensions.swift - Copyright 2023 SwifterSwift
+
+#if canImport(Foundation)
+import Foundation
+
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
+
+// MARK: - Methods
+
+public extension URLSession {
+    /// SwifterSwift: synchronious version of `func dataTask(with request: URLRequest, completionHandler: @escaping @Sendable (Data?, URLResponse?, Error?) -> Void) -> URLSessionDataTask`
```

Comment:
This is not a complete code entry. In the Xcode menu select Editor -> Structure -> Add Documentation and you'll get:
```swift
/// <#Description#>
/// - Parameter request: <#request description#>
/// - Throws: <#description#>
/// - Returns: <#description#>
```

Reply:
Hello @guykogus 
Please check this function again.